src/api/public/apidocs-new/paths/person.yaml

```
@@ -38,3 +38,35 @@ get:
       $ref: '../components/responses/unauthorized.yaml'
   tags:
     - Person
+
+post:
+  summary: Executes a command on the person endpoint.
+  description: >-
+    Allows executing command on the person endpoint.
+    As of now, the only command allowed is 'register'.
```

```suggestion
    Allows executing command on the person endpoint.
    
    As of now, the only command allowed is 'register'.
```
These two lines show as one in the Swagger UI. Adding an empty line between them will fix it.